File: Source/DroolTool.Web/src/app/shared/models/neighborhood-metric-dto.ts
export class NeighborhoodMetricDto{
    public MetricYear: number;
    public MetricMonth: number;
    public TotalDrool?: number;
    public OverallParticipation?: number;
    public PercentParticipation?: number;
    public DroolPerLandscapedAcre?: number;
    public TotalWaterAccounts?: number;
    public TotalIrrigatedArea?: number;
    public TotalWaterUsedForIrrigation?: number;
    public HoaWaterUsedForIrrigation?: number;
    public DroolPerLandscapedAcreYearlyPercentDifference?: number;

    constructor(obj?: any){
        Object.assign(this, obj);
    }
}

File: DroolTool.Models/DataTransferObjects/NeighborhoodMetricDto.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace DroolTool.Models.DataTransferObjects
{
    public class NeighborhoodMetricDto
    {
        public int MetricYear { get; set; }
        public int MetricMonth { get; set; }
        public double? TotalDrool { get; set; }
        public double? OverallParticipation { get; set; }
        public double? PercentParticipation { get; set; }
        public double? DroolPerLandscapedAcre { get; set; }
        public double? TotalWaterAccounts { get; set; }
        public double? ResidentialWaterAccounts { get; set; }
        public double? HOAWaterAccounts { get; set; }
        public double? CommercialWaterAccounts { get; set; }
        public double? MunicipalWaterAccounts { get; set; }
        public double? TotalIrrigatedArea { get; set; }
        public double? TotalWaterUsedForIrrigation { get; set; }
        public double? HoaWaterUsedForIrrigation { get; set; }
        public double? DroolPerLandscapedAcreYearlyPercentDifference { get; set; }
    }
}
